qdrant-landing/content/documentation/frameworks/pandas-ai.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Pandas-AI
+weight: 2900
+---
+
+# Pandas-AI
+
+Pandas-AI is a Python library that uses a generative AI model to interpret natural language queries and translate them into Python code to interact with the data and return the results to the user.
```

Comment:
```suggestion
Pandas-AI is a Python library that uses a generative AI model to interpret natural language queries and translate them into Python code to interact with pandas data frames and return the final results to the user.
```

I was a bit confused if the names of `pandas-ai` and `pandas` are somehow related to each other.